sample-apps/java-sample-app/manual/src/main/java/software/amazon/adot/sampleapp/ManualApp.java

```diff
@@ -72,24 +76,24 @@ private static OpenTelemetry buildOpentelemetry() {
                 .setTracerProvider(
                         SdkTracerProvider.builder()
                                 .addSpanProcessor(
-                                        BatchSpanProcessor.builder(OtlpGrpcSpanExporter.getDefault()).build())
+                                        BatchSpanProcessor.builder(OtlpGrpcSpanExporter.builder().setEndpoint(exporter).build()).build())
                                 .setIdGenerator(AwsXrayIdGenerator.getInstance())
                                 .setResource(resource)
                                 .build())
                 .setMeterProvider(
                         SdkMeterProvider.builder()
                                 .registerMetricReader(
-                                        PeriodicMetricReader.builder(OtlpGrpcMetricExporter.getDefault()).build())
+                                        PeriodicMetricReader.builder(OtlpGrpcMetricExporter.builder().setEndpoint(exporter).build()).setInterval(1, TimeUnit.SECONDS).build())
```

Comment:
Are we setting exporting interval of 1s in the other languages? If not, we should just use the default, and there is no need to use `setInterval`.

Reply:
We are not so I have remove the setinterval in my latest commit.